h/browser/chrome/lib/options.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Hypothesis Extension</title>
+  <style>
+    .hint {
+      font-size: 90%;
+    }
+  </style>
+</head>
+
+<body>
+  <input type="checkbox" id="badge">
+  <label for="badge">Enable extension badge?</label>
+  <p class="hint">
+      By default, the extension button has a small badge that displays the
```

Comment:
I'm not really sure I understand. This isn't about permissions, it's about the badge. Linking to the permissions blog post is just going to be confusing.